newton/tests/test_body_force.py

```
@@ -228,6 +310,656 @@ def test_3d_articulation(test: TestBodyForce, device, solver_fn, test_angular, u
+        displacement,
+        0.001,
+        "Body should have translated but position barely changed",
+    )
+
+
+def test_combined_force_torque_control_joint_f(
+    test: TestBodyForce,
+    device,
+    solver_fn,
+    com_offset: tuple[float, float, float],
+    tolerance: float,
```

Ruff (0.14.11)

Unused function argument: `tolerance`

(ARG001)